### PR TITLE
Wrpd enhancement 32

### PR DESCRIPTION
**Pull request by irkfaisal**
## Description

Add label/title before packages details. 

## Checklist

- [ ] I have performed a thorough self-review of my code.
- [ ] I have added or updated relevant tests for my changes.
- [ ] My code follows the project's style guidelines and best practices.
- [ ] I have updated the documentation if necessary.
- [ ] I have added or updated relevant comments in my code.
- [ ] I have resolved any merge conflicts of my branch.

## Reviewers

---

## Maintainer Notes

- [ ] Has this change been tested in a staging environment?
- [ ] Does this change introduce any breaking changes or deprecations?

### CODE REVIEW

**Review by techoneel — CHANGES_REQUESTED** [review]
I do not like the architecture of the changes requested.

Please have a discussion with technical team and clear your doubts what was wrong and work on it and fix it.

**Comment by irkfaisal** [comment]
I have changed the architecture and committed again kindly review.

File: src/scripts/web-runtime-core-package.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (techoneel) [inline src/scripts/web-runtime-core-package.json:1]: Remove unnecessary json files

**Review by techoneel — CHANGES_REQUESTED** [review]
Remove unnecessary json files

**Pull request merged**